src/services/database.js

```diff
@@ -264,3 +251,69 @@ export async function setTriviaHints(id, hints) {
     throw err;
   }
 }
+
+export async function getTriviaHintLevel() {
+  try {
```

Comment:
If you're not handling an error, you don't need to re-throw the error, just omit the try/catch and the error bubbles up